salt/modules/state.py

```diff
@@ -256,14 +256,22 @@ def _get_opts(**kwargs):
     '''
     Return a copy of the opts for use, optionally load a local config on top
     '''
-    opts = copy.deepcopy(__opts__)
     if 'localconfig' in kwargs:
         opts = salt.config.minion_config(kwargs['localconfig'], defaults=opts)
```

Comment:
I squashed my correction into the prior commit, as I don't like leaving broken code in intermediate commits during a PR. It can mess with bisects.